mara_pipelines/execution.py

```diff
@@ -53,6 +53,11 @@ def run_pipeline(pipeline: pipelines.Pipeline, nodes: {pipelines.Node} = None,
     # A queue for receiving events from forked sub processes
     event_queue = multiprocessing_context.Queue()
 
+    # The historical node cost is taken from the 'mara' db. If the 'mara' datatabase is not defined
+    # (e.g. in testing scenarios) we do not use the node cost.
+    import mara_db.config
+    use_historical_node_cost = 'mara' in mara_db.config.databases()
+
```

Comment:
I would add a big warning to stderr here to make ppl aware on regular scenarious when you (usually?) want a mara db...

Reply:
Done